core/src/main/java/com/novoda/noplayer/internal/exoplayer/NoPlayerAdsLoader.java

```diff
@@ -150,6 +159,11 @@ public void release() {
     public void handlePrepareError(int adGroupIndex, int adIndexInAdGroup, IOException exception) {
         if (adPlaybackState != null) {
             adPlaybackState = adPlaybackState.withAdLoadError(adGroupIndex, adIndexInAdGroup);
+            updateAdPlaybackState();
```

Comment:
Yea we actually need this 😂 if we don't do this then the `AdPlaybackState` is marked with an error but is never passed to the underlying `source` and so playback just stops. No advert events, nothing 😂